Open Judge System/Web/OJS.Web/App_Start/NinjectWebCommon.cs

```diff
@@ -79,7 +78,16 @@ private static void RegisterServices(IKernel kernel)
             kernel.Bind<ISimilarityFinder>().To<SimilarityFinder>().InRequestScope();
             kernel.Bind<IPlagiarismDetectorFactory>().To<PlagiarismDetectorFactory>().InRequestScope();
             kernel.Bind<IHangfireBackgroundJobService>().To<HangfireBackgroundJobService>().InRequestScope();
-            kernel.Bind<ISubmissionsForProcessingDataService>().To<SubmissionsForProcessingDataService>().InRequestScope();
+
+            kernel
```

Comment:
We should take note that as we implement more and more services it would become increasingly necessary to extract an entire separate Ninject module.

Reply:
Maybe the next issue I have to work on is this: https://github.com/SoftUni-Internal/suls-issues/issues/3212